### PR TITLE
[cdc] Add submodules

### PR DESCRIPTION
**Pull request by eunchan**
Add submodules and generate report per module.

### CODE REVIEW

**Review by tjaychen — APPROVED** [review]

File: hw/cdc/tools/verixcdc/run-cdc.tcl [hunk 1]
```diff
@@ -108,3 +108,13 @@ source $CDC_WAIVER_FILE
 #########################
 
 report_policy -verbose -skip_empty_summary_status -compat -output vcdc.rpt ALL
+
+set modules {spi_device kmac hmac uart gpio spi_host flash_ctrl alert_handler otp_ctrl lc_ctrl pwrmgr clkmgr rstmgr keymgr csrng entropy_src aes}
```
Comment (msfschaffner) [inline hw/cdc/tools/verixcdc/run-cdc.tcl:112]: That is great! We should probably enhance our dashboard at some point to parse the reports out correctly.

File: hw/top_earlgrey/cdc/cdc_waivers.gpio.tcl [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Verix CDC waiver file
+
+set_rule_status -rule { W_RECON_GROUPS } -status { Waived } \
+    -expression {u_gpio.gen_filter} \
+   -comment {filters are reconverged into registers. But it's OK.}
```
Comment (msfschaffner) [inline hw/top_earlgrey/cdc/cdc_waivers.gpio.tcl:9]: jsut a nit: alignment and missing endline.

**Review by msfschaffner — APPROVED** [review]
LGTM!

File: hw/cdc/tools/verixcdc/run-cdc.tcl (hunk 1 above)
Comment (eunchan) [reply hw/cdc/tools/verixcdc/run-cdc.tcl]: One issue is that the tool changes the module names after elaboration if the module is instantiated multiple times (e.g. uart/ spi_host). Need to resolve this.

**Pull request merged**